### PR TITLE
Update help me choose to include ADX

### PR DESCRIPTION
**Pull request by arthurclares**
## 🛠️ Description
This PR adds information about Hubs on ADX into the Help Me choose section.

Fixes # 

### 🔬 How did you test this change?

> - [ ] 🤏 Lint tests
> - [ ] 🤞 PS -WhatIf / az validate
> - [x] 👍 Manually deployed + verified
> - [ ] 💪 Unit tests
> - [ ] 🙌 Integration tests

### 🙋‍♀️ Do any of the following that apply?

> - [ ] 🚨 This is a breaking change.
> - [x] 🤏 The change is less than 20 lines of code.

### 📑 Did you update `docs/changelog.md`?

> - [ ] ✅ Updated changelog (required for `dev` PRs)
> - [ ] ➡️ Will add log in a future PR (feature branch PRs only)
> - [x] ❎ Log not needed (small/internal change)

### 📖 Did you update documentation?

> - [x] ✅ Public docs in `docs` (required for `dev`)
> - [ ] ✅ Internal dev docs in `src` (required for `dev`)
> - [ ] ➡️ Will add docs in a future PR (feature branch PRs only)
> - [ ] ❎ Docs not needed (small/internal change)

### CODE REVIEW

File: docs/_reporting/hubs/README.md [hunk 1]
```diff
@@ -38,10 +38,23 @@ FinOps hubs are a reliable, trustworthy platform for cost analytics, insights, a
...
+
+  Hubs with Azure Data Explorer:  ** ~$220 per month**
+    _Estimated cost includes The estimated cost of Hubs on ADX assumes instance is running 24x7. Costs can be reduced by enabling auto-shutdown outside of business hours or applying commitment discounts. Prices are estiamted in UK South as of October 2024. If you are using Power BI dashboards, you need to include up to $20 per user for [Power BI licenses](https://www.microsoft.com/power-platform/products/power-bi/pricing). Exact cost will vary based on discounts, data size per $1M (~20GB of data), and Power BI license requirements. Pipelines will run once a day per export, plus one additional monthly run per export. Pipeline run time depends on data size. For details, refer to the [FinOps hub cost estimate](https://aka.ms/finops/hubs/calculator/adx) in the Azure Pricing Calculator or monitor hub cost using the [Data ingestion report](../power-bi/data-ingestion.md)._
```
Comment (flanakin) [inline docs/_reporting/hubs/README.md:57]: Fwiw, the previous block of text was already too big. This is making it even bigger. Should we move this to a section further down? It's not enough for a separate doc.

File: docs/_reporting/hubs/README.md [hunk 2]
```diff
@@ -82,13 +95,15 @@ _<sup>2) EA billing scopes can be exported to any tenant today. Simply sign in t
 
 ## 📦 What's included
 
-The FinOps hub template includes the following resources:
+The default deployment of FinOps hub template includes the following resources:
 
 - Storage account (Data Lake Storage Gen2) to hold all cost data.
 - Data Factory instance to manage data ingestion and cleanup.
 - Key Vault to store the Data Factory system managed identity credentials.
```
Comment (flanakin) [inline docs/_reporting/hubs/README.md:102]: I would revert this and just add ADX to the list and call out that it's optional.
```suggestion
The FinOps hub template includes the following resources:

- Data Factory instance to manage data ingestion and cleanup.
- Storage account (Data Lake Storage Gen2) as a staging area for data ingestion.
- Azure Data Explorer (Kusto) as a scalable datastore for advanced analytics (optional).
- Key Vault to store the Data Factory system managed identity credentials.
```

File: docs/_reporting/hubs/README.md [hunk 3]
```diff
@@ -82,13 +95,15 @@ _<sup>2) EA billing scopes can be exported to any tenant today. Simply sign in t
...
 
-Once deployed, you can report on the data in Power BI or by connecting to the storage account directly.
+If you would like to use the new Azure Data Explorer features, you will need to enbale the parameter **dataExplorerName**. To learn more about the parameters needed to deploy Hubs with ADX, refer to the [template](../hubs/template.md) page.
+
```
Comment (flanakin) [inline docs/_reporting/hubs/README.md:105]: I don't think this is needed here. We're just explaining what's in the template at a high level. We shouldn't get into the details of how to enable features.
```suggestion
```

File: docs/_reporting/hubs/README.md [hunk 4]
```diff
@@ -82,13 +95,15 @@ _<sup>2) EA billing scopes can be exported to any tenant today. Simply sign in t
...
+Once deployed, you can report on the data in ADX dashboards (__comming soon__), Power BI or by connecting to the storage account directly.
```
Comment (flanakin) [inline docs/_reporting/hubs/README.md:106]: Technically, they can use ADX dashboards themselves.
```suggestion
Once deployed, you can report on the data directly using Data Explorer queries, Data Explorer dashboards, Power BI, or by connecting to the database or storage account directly.
```

File: docs/_reporting/hubs/README.md [hunk 5]
```diff
@@ -82,13 +95,15 @@ _<sup>2) EA billing scopes can be exported to any tenant today. Simply sign in t
...
 
 <img alt="Screenshot of the cost summary report" style="max-width:200px" src="https://user-images.githubusercontent.com/399533/216882658-45f026f1-c895-48ca-81e2-35765af8e29e.png">
 <img alt="Screenshot of the services cost report" style="max-width:200px" src="https://user-images.githubusercontent.com/399533/216882700-4e04b589-0580-4e49-9b40-9f5948792975.png">
```
Comment (flanakin) [inline docs/_reporting/hubs/README.md:109]: I can't leave any comments below this point since nothing changed, but we should probably also update:
1. Should we explicitly call out the ADX cluster name in the Create a new hub > Deploy your FinOps hub section? That's where we want them to make sure to enable it.
2. Connect to your data should be updated to call out ADX.
3. Get started with hubs should also call out ADX.
4. The required permissions below aren't the full story, but we should confirm if the template.md file has the required permissions. I don't think I did that.

File: docs/_reporting/power-bi/README.md [hunk 6]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
 
 When you need more advanced reporting or to merge with your own data, we recommend using Microsoft Fabric, Power BI, or a custom or third-party solution. Use the following to determine the best approach for you:
 
-1. For costs under $2-5M in total<sup>1</sup> that don't need savings plan data, you can use the Cost Management connector for Power BI.
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:60]: I think we need to keep this. It's still a reality and default expectation for many customers. We need to convey why it's not a good idea.

File: docs/_reporting/power-bi/README.md [hunk 7]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
-   - The connector uses existing raw cost data APIs and cannot scale to data sizes beyond $5M<sup>1</sup>.
-   - Due to the size constraints, the connector will be phased out by the Cost Management team starting in 2024.
-   - The APIs do not include some key columns for savings plans, like the BenefitId/Name columns. All costs are covered but not always easily identifiable.
-2. For costs under $2-5M/month (~$65M total)<sup>2</sup> that need savings plan data, you can use raw exports with Power BI.
-3. For costs under $2-5M/month (~$65M total)<sup>2</sup> that need savings plan data, you can use FinOps hubs with Power BI.
+1. **Power BI on Cost Exports**: For costs under $2-5M/month (~$65M total)<sup>2</sup> that need savings plan data, you can use raw exports with Power BI.
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:60]:
```suggestion
1. **Power BI on Cost Management exports**: For costs under $2-5M/month (~$65M total)<sup>2</sup> that need savings plan data, you can use raw exports with Power BI.
```

File: docs/_reporting/power-bi/README.md [hunk 8]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
+2. **FinOps Hubs with Azure Data Factory**: For costs under $2-5M/month (~$65M total)<sup>2</sup> that need savings plan data, you can use FinOps hubs with Power BI.
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:61]:
```suggestion
2. **FinOps hubs on storage**: For costs under $2-5M/month (~$65M total)<sup>2</sup> that require support for multiple Entra ID tenants or prefer the solution to manage exports for you, you can use FinOps hubs with Power BI.
```

File: docs/_reporting/power-bi/README.md [hunk 9]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
+3. 🆕 **FinOps Hubs with Azure Data Explorer (ADX)**:  For scenarios where you need more advanced exploration, Hubs on ADX allows you to leverage Kusto Query Language (KQL) for in-depth analysis and provides built-in queries to simplify data exploration. ADX also offers scalability, allowing you to increase capacity by adjusting the virtual machine size. The default VM size supports up to $10M/month of cost data.
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:62]: I don't think we need to get into sizing here. We just need to convey why they should choose this option over one before it.
```suggestion
3. 🆕 **FinOps hubs with Azure Data Explorer**:  For costs over $2-5M/month or when you need more advanced exploration, we recommend FinOps hubs with Azure Data Explorer (ADX). Leverage Kusto Query Language (KQL) for advanced analytics and improved performance and scalability.
```

File: docs/_reporting/power-bi/README.md [hunk 10]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
 4. _**Coming soon:**_ For costs over $5M/month or for additional capabilities, you can connect Fabric to either FinOps hubs or raw exports.
    - This is possible today, but is not supported in FinOps toolkit reports yet. Support will be added in a future release.
 
 _<sup>1) Power BI Pro can handle ~$2M of raw cost data. Power BI Premium can handle ~$5M.</sup>_
 
 _<sup>2) The $2-5M limits are for Power BI data refreshes and apply on a monthly basis for hubs and raw exports. They can load up to $65M with incremental refresh enabled.</sup>_
 
-In general, we recommend starting with the Cost Management connector when getting started with Power BI reports. The most common reasons to switch to FinOps hubs are for additional account types and scopes or to enable more advanced capabilities. Use the following comparison to help you make the decision:
+_<sup>3) ❗ The Cost Management connector is no longer recommended as it will be deprecated soon.</sup>_
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:70]: This isn't referenced anywhere. Not sure we need it if we're explicitly calling out what we do recommend. If we use something like this, we should keep it with the section above.
```suggestion
```

File: docs/_reporting/power-bi/README.md [hunk 11]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
 
 When you need more advanced reporting or to merge with your own data, we recommend using Microsoft Fabric, Power BI, or a custom or third-party solution. Use the following to determine the best approach for you:
 
-1. For costs under $2-5M in total<sup>1</sup> that don't need savings plan data, you can use the Cost Management connector for Power BI.
-   - The connector uses existing raw cost data APIs and cannot scale to data sizes beyond $5M<sup>1</sup>.
-   - Due to the size constraints, the connector will be phased out by the Cost Management team starting in 2024.
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:62]: We should change this to 2025

File: docs/_reporting/power-bi/README.md [hunk 12]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
 
 _<sup>1) Power BI Pro can handle ~$2M of raw cost data. Power BI Premium can handle ~$5M.</sup>_
 
 _<sup>2) The $2-5M limits are for Power BI data refreshes and apply on a monthly basis for hubs and raw exports. They can load up to $65M with incremental refresh enabled.</sup>_
 
-In general, we recommend starting with the Cost Management connector when getting started with Power BI reports. The most common reasons to switch to FinOps hubs are for additional account types and scopes or to enable more advanced capabilities. Use the following comparison to help you make the decision:
+_<sup>3) ❗ The Cost Management connector is no longer recommended as it will be deprecated soon.</sup>_
 
-| Capabilities                                        |            Connector             |             Exports              |           FinOps hubs            | Microsoft Fabric<sup>1</sup> |
+In general, we recommend starting with Power BI reports by connecting to the cost exports. The most common reasons to switch to FinOps hubs are for additional account types and scopes or to enable more advanced capabilities. Use the following comparison to help you make the decision:
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:72]:
```suggestion
In general, we recommend starting with Power BI reports by connecting to the Cost Management exports. The most common reasons to switch to FinOps hubs are for performance, scale, and to enable more advanced capabilities. Use the following comparison to help you make the decision:
```

File: docs/_reporting/power-bi/README.md [hunk 13]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
+
+| Capabilities                                        |             Power BI on Cost Exports              |        FinOps hubs with ADF         |        FinOps hubs with ADF and ADX         | Microsoft Fabric<sup>1</sup> |
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:74]:
```suggestion
| Capabilities                                        |             Exports              |        FinOps hubs (storage)         |        FinOps hubs (ADX)         | Microsoft Fabric<sup>1</sup> |
```

File: docs/_reporting/power-bi/README.md [hunk 14]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
-| Faster data load times                              |                ❌                 |                🔜                 |                ✅                 |         ✅ (via Hubs)         |
-| Supports >$65M in cost details                      |                ❌                 |                ❌                 |             🔜 (0.7)              |              ✅               |
-| Analytical engine                                   |                ❌                 |                ❌                 |             🔜 (0.7)              |              ✅               |
-| Accessible outside of Power BI                      |                ❌                 |                ✅                 |                ✅                 |              ✅               |
-| Learn more                                          |  [Learn more][about-connector]   |                                  |     [Learn more][about-hubs]     |                              |
+| Cost (based on list prices)                         |           ~$10 per $1M           |           ~$25 per $1M           |      ~$220 per month<sup>2</sup>  |             TBD              |
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:76]: Let's use the dev/test pricing, since that's default and all most people will need.
```suggestion
| Monthly cost (based on list prices)                 |           ~$23 per $1M           |           ~$25 per $1M           |      $120 + ~$30 per $1M<sup>2</sup>  |             $300+              |
```

File: docs/_reporting/power-bi/README.md [hunk 15]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
+| Data storage                                        |        Data Lake Storage         |        Data Lake Storage         | Data Lake Storage + Azure Data Explorer |      Data Lake Storage       |
+| Est. max raw cost details per month<sup>3</sup>     | $2M/mo (Pro)<br>$5M/mo (Premium) | $2M/mo (Pro)<br>$5M/mo (Premium) |                                  |             TBD              |
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:78]:
```suggestion
| Est. max raw cost details per month<sup>3</sup>     | $2M/mo (Pro)<br>$5M/mo (Premium) | $2M/mo (Pro)<br>$5M/mo (Premium) |              TBD                 |             TBD              |
```

File: docs/_reporting/power-bi/README.md [hunk 16]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
+| Est. max total with incremental refresh<sup>4</sup> |   $2M (Pro)<br>$65M (Premium)    |   $2M (Pro)<br>$65M (Premium)    |                                  |             TBD              |
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:79]:
```suggestion
| Est. max total with incremental refresh<sup>4</sup> |   $2M (Pro)<br>$65M (Premium)    |   $2M (Pro)<br>$65M (Premium)    |              TBD                 |             TBD              |
```

File: docs/_reporting/power-bi/README.md (hunk 13 above)
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:74]: The further down this list I get, the more I wonder if we should even call out hubs on storage 🤔

File: docs/_reporting/power-bi/README.md [hunk 17]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
+| Microsoft Online Services Agreement                 |                ❌                 |                ❌                 |             ❌                     |              ❌               |
+| Billing accounts                                    |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Billing profiles                                    |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Invoice sections                                    |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| CSP customers (partner only)                        |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Management groups                                   |                ❌                 |                ❌                 |             ✅                     |              ❌               |
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:91]:
```suggestion
| Management groups                                   |                ❌                 |                ❌                 |             ❌                     |              ❌               |
```

File: docs/_reporting/power-bi/README.md [hunk 18]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
+| Subscriptions                                       |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Resource groups                                     |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Supports savings plans<sup>4</sup>                  |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Supports savings plan recommendations               |                ❌                 |                🔜                 |             ✅                     |              🔜               |
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:95]:
```suggestion
| Supports savings plan recommendations               |                ❌                 |                🔜                 |             🔜                     |              🔜               |
```

File: docs/_reporting/power-bi/README.md [hunk 19]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
    - This is possible today, but is not supported in FinOps toolkit reports yet. Support will be added in a future release.
 
 _<sup>1) Power BI Pro can handle ~$2M of raw cost data. Power BI Premium can handle ~$5M.</sup>_
 
 _<sup>2) The $2-5M limits are for Power BI data refreshes and apply on a monthly basis for hubs and raw exports. They can load up to $65M with incremental refresh enabled.</sup>_
 
-In general, we recommend starting with the Cost Management connector when getting started with Power BI reports. The most common reasons to switch to FinOps hubs are for additional account types and scopes or to enable more advanced capabilities. Use the following comparison to help you make the decision:
+_<sup>3) ❗ The Cost Management connector is no longer recommended as it will be deprecated soon.</sup>_
 
-| Capabilities                                        |            Connector             |             Exports              |           FinOps hubs            | Microsoft Fabric<sup>1</sup> |
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:75]: After getting thru the table, I realized you got rid of the connector. We need to keep that.

File: docs/_reporting/power-bi/README.md [hunk 20]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
+| Subscriptions                                       |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Resource groups                                     |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Supports savings plans<sup>4</sup>                  |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Supports savings plan recommendations               |                ❌                 |                🔜                 |             ✅                     |              🔜               |
+| Supports multiple scopes                            |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Supports scopes in different tenants<sup>5</sup>    |          ⚠️<sup>5</sup>           |          ⚠️<sup>5</sup>           |             ❌                     |          ⚠️<sup>5</sup>(via Hubs) |
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:97]:
```suggestion
| Supports scopes in different tenants<sup>5</sup>    |          ❌<sup>5</sup>           |          ✅           |             ✅                     |          ✅ (via Hubs) |
```

File: docs/_reporting/power-bi/README.md [hunk 21]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
+| Faster data load times                              |                🔜                 |                ✅                 |             ✅                     |         ✅ (via Hubs)         |
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:98]:
```suggestion
| Faster data load times                              |                ❌                 |                ❌                 |             ✅                     |         ✅         |
```

File: docs/_reporting/power-bi/README.md [hunk 22]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
+| Supports >$65M in cost details                      |                ❌                 |                ✅                 |             ❌                     |              ✅               |
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:99]:
```suggestion
| Supports >$65M in cost details                      |                ❌                 |                ❌                 |             ✅                     |              ✅               |
```

File: docs/_reporting/power-bi/README.md [hunk 23]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
+| Accessible outside of Power BI                      |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Kusto Query Language (KQL) support                        |                ❌                 |                ❌                 |             ✅                     |              ❌               |
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:101]:
```suggestion
| Kusto Query Language (KQL) support                        |                ❌                 |                ❌                 |             ✅                     |              ✅               |
```

File: docs/_reporting/power-bi/README.md [hunk 24]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
+| Native integration with cost optimization workbook |         ❌                 |                ❌                 |             ✅<sup>6</sup>                     |              ❌               |
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:102]: Should we expand this to workbooks in general? Then that's possible today. It's just not something we offer. Beyond that, I'm not sure the COW is big enough to really call out as a feature someone's going to use to make a decision 🤔
```suggestion
| Native integration with cost optimization workbook |         ❌                 |                ❌                 |             🔜                     |              ❌               |
```

File: docs/_reporting/power-bi/README.md [hunk 25]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
+| Azure Data Explorer built-in dashboards                    |                ❌                 |                ❌                 |             ✅                     |              ❌               |
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:103]: This doesn't feel applicable in a Power BI doc.
```suggestion
```

File: docs/_reporting/power-bi/README.md [hunk 26]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
 
 [about-connector]: https://aka.ms/costmgmt/powerbi
 [about-hubs]: ../hubs/README.md
+[about-rawexports]: ../power-bi/setup.md
 [hubs-template]: ../hubs/template.md
 
 _<sup>1) Microsoft Fabric can connect to either raw exports or FinOps hubs. FinOps toolkit reports do not support Microsoft Fabric yet but will in a future release.</sup>_
 
-_<sup>2) Power BI constraints are based on data size and processing time. Monitored spend estimations are for reference only. You may see different limits based on services you use and other datasets you ingest.</sup>_
+<sup>2) The estimated cost of Hubs on ADX assumes the two instances are running 24x7. Costs can be reduced by enabling auto-shutdown outside of business hours or applying commitment discounts. Prices are estiamted in UK South as of October 2024</sup>_
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:113]: Can you reword this to say that the estimate assumes a single node dev/test cluster and that they may need to consider additional nodes for larger accounts? I don't know what the precise guidance should be, so we'll have to be hand-wavy.

File: docs/_reporting/power-bi/README.md [hunk 27]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
 
-_<sup>3) The Cost Management connector for Power BI does not support incremental refresh, so the limits are the same as the per-month estimation. The FinOps hub estimate is based on incremental refresh being enabled, which requires additional configuration after your report is published.</sup>_
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:114]: Bring this back.

File: docs/_reporting/power-bi/README.md [hunk 28]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
+_<sup>3) Power BI constraints are based on data size and processing time. Monitored spend estimations are for reference only. You may see different limits based on services you use and other datasets you ingest.</sup>_
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:115]: I like this note, but is it referenced above? It might belong somewhere else.

File: docs/_reporting/power-bi/README.md [hunk 29]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
 
-_<sup>4) The Cost Management connector uses an old API version and does not include details for some features, like savings plans. Please use FinOps hubs for the latest version with all details.</sup>_
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:116]: Bring this back.

File: docs/_reporting/power-bi/README.md [hunk 30]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
+_<sup>4) The FinOps hub estimate is based on incremental refresh being enabled, which requires additional configuration after your report is published.</sup>_
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:117]: Where is this referenced? I'm not sure it's needed since we have a reference to incremental refresh earlier.

File: docs/_reporting/power-bi/README.md [hunk 31]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
 
 _<sup>5) EA billing scopes can be exported to any tenant today. Simply sign in to that tenant with an account that has access to the billing scope and target storage account to configure exports. Non-billing scopes (subscriptions, management groups, and resource groups) and all MCA scopes are only supported in the tenant they exist in today but will be supported via a "remote hubs" feature in a future FinOps hubs release.</sup>_
 
-If you're not sure, start with the Cost Management connector. You will usually be able to tell if that works for you within the first 5-10 minutes. If you experience delays in pulling your data, try requesting fewer months. If you still experience issues, it's time to consider switching to FinOps hubs.
+_<sup>6) _Comming soon_. Dashboards will be available in a future release.</sup>_
+
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:122]: Not needed. The icon conveys this already.
```suggestion
```

File: docs/_reporting/power-bi/README.md [hunk 32]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
+If you're unsure where to start, we recommend downloading the Power BI dashboards and connecting them to cost exports. This will allow you to explore the reports and see how they work with your data. Alternatively, you can open the Power BI reports using the provided sample data.
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:123]:
```suggestion
If you're unsure where to start, we recommend downloading the Power BI dashboards and connecting them to Cost Management exports in storage. This will allow you to explore the reports and see how they work with your data. Alternatively, you can open the Power BI reports using the provided sample data.

```

File: docs/_reporting/power-bi/README.md [hunk 33]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
+Our recommendation is to use Hubs on ADX, as it offers exclusive features not available in other versions. These include querying cost data with KQL and native integration with the Cost Optimization workbook. However, if you don't require any of ADX's exclusive capabilities, you can opt for Hubs on ADF instead.
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:124]: Not sure we need to say some of this again.
```suggestion
For the best performance and capabilities, we recommend using FinOps hubs with Data Explorer, as it offers exclusive features not available in other options listed above.
```

File: docs/_reporting/power-bi/README.md [hunk 34]
```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
...
+| Invoice sections                                    |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| CSP customers (partner only)                        |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Management groups                                   |                ❌                 |                ❌                 |             ✅                     |              ❌               |
+| Subscriptions                                       |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Resource groups                                     |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Supports savings plans<sup>4</sup>                  |                ✅                 |                ✅                 |             ✅                     |              ✅               |
```
Comment (flanakin) [inline docs/_reporting/power-bi/README.md:94]:
```suggestion
| Calculate EA and MCA cost savings                  |                ❌                 |                ❌                 |             ✅                     |              ✅ (via Hubs with ADX)               |
| Supports savings plans<sup>4</sup>                  |                ✅                 |                ✅                 |             ✅                     |              ✅               |
```

File: docs/_reporting/power-bi/README.md (hunk 25 above)
Comment (arthurclares) [reply docs/_reporting/power-bi/README.md:103]: THat is why I suggested this doc to be somewhere else. This is a help me choose section for Hubs.

Comment (flanakin) [reply docs/_reporting/power-bi/README.md:104]: But this isn't about hubs. Right now it's about the options available for using Power BI. If we want to expand it to include other reporting options, then those would need to be columns and not rows. I would remove this for now since we don't even have any dashboards available out of the box.

File: docs/_reporting/hubs/README.md [hunk 35]
```diff
@@ -38,10 +38,23 @@ FinOps hubs are a reliable, trustworthy platform for cost analytics, insights, a
 
 We are very early in our journey. Today, FinOps hubs extend Cost Management by exporting cost details to a consolidated storage account and addressing a few of the inherent limitations that make exports more difficult to use. In their most basic form, FinOps hubs enable more Power BI reporting options. On the more advanced end, FinOps hubs are a foundation for you to build your own cost management and optimization solution.
```
Comment (flanakin) [inline docs/_reporting/hubs/README.md:39]:
```suggestion
FinOps hubs extend Cost Management to provide a scalable platform for advanced data reporting and analytics, through tools like Power BI and Microsoft Fabric. FinOps hubs are a foundation to build your own cost management and optimization solution.
```
